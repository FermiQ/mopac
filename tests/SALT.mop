geo_dat="Neutral2.mop"  OUTPUT   site=(salt)
Test that salt bridges are generated correctly


