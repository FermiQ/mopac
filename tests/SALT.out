
          GEO_DAT file: 'Neutral.mop' does not exist.

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * GEO_DAT file: 'Neutral.mop' does not exist.                                *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
